### PR TITLE
Update Release notes for Preview Release v1.0.19189.1

### PR DESCRIPTION
**Pull request by cheenamalhotra**

### CODE REVIEW

File: release-notes/1.0/1.0.19189.1-Preview.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+* System.Security.Principal.Windows 4.5.0
+* System.Text.Encoding.CodePages 4.5.0
+* System.Configuration.ConfigurationManager 4.5.0
+
+### Fixed
+
+* Microsoft.SqlServer.Server conflicts between System.Data.SqlClient and Microsoft.Data.SqlClient [SqlClient Issue #4](https://github.com/dotnet/SqlClient/issues/4)
+  SqlDataRecord, SqlMetaData, and UDT related classes were moved into Microsoft.Data.SqlClient.Server to avoid namespace conflicts. While UDT related classes are there, UDTs do not currently work in Microsoft.Data.SqlClient.
+* Committing certain data results in an infinite loop inside System.Data.SqlClient.TdsParserStateObject.cs. Ported from [CoreFx PR 34709](https://github.com/dotnet/corefx/pull/34709)
+* When targeting **.NET Framework** and platform **AnyCPU** or **x86**, you may see a build warning about processor architecture mismatch: *Warning	There was a mismatch between the processor architecture of the project being built "MSIL" and the processor architecture of the reference "Microsoft.Data.SqlClient*...
```
Comment (peterbae) [inline release-notes/1.0/1.0.19189.1-Preview.md]: Maybe this line needs a bit of rewording, with the tab spacing after warning and "Microsoft.Data.SqlClient*

File: release-notes/1.0/1.0.19189.1-Preview.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+.NET Framework
+
+* Fixed an issue when a pooled connection was switched between databases supporting UTF-8 which resulted in a mismatch between encoding and code page.
+
+.NET Core
+
+* When doing a hard copy of a sql parameter, the MetaType was not copied if the value of the parameter was not defined. This resulted in the wrong SqlType of the parameter.
+* Fixed in 1.0.19123.2-Preview
+  * XEvents are arriving as expected, but with 1 XEvent delay [CoreFx PR 36015](https://github.com/dotnet/corefx/pull/36015)
```
Comment (peterbae) [inline release-notes/1.0/1.0.19189.1-Preview.md]: make spacing consistent

File: release-notes/1.0/1.0.19189.1-Preview.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+.NET Core changes
+
+* Enable BeginExecuteReader methods on SqlCommand from [CoreFx PR 33660](https://github.com/dotnet/corefx/pull/33660)
+
+.NET Core performance improvements
+
+* Avoid unnecessary Span to ReadOnlySpan casts from [CoreFx PR 34646](https://github.com/dotnet/corefx/pull/34646)
+* SqlClient environment change optimization from [CoreFx PR 34573](https://github.com/dotnet/corefx/pull/34573)
+* Changes in 1.0.19123.2 Preview
+  * Memory allocation enhancement from [CoreFx PR 34047](https://github.com/dotnet/corefx/pull/34047)
```
Comment (peterbae) [inline release-notes/1.0/1.0.19189.1-Preview.md]: make spacing consistent

File: release-notes/1.0/1.0.19189.1-Preview.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+    {
+        public Microsoft.Data.SqlClient.DataClassification.InformationType InformationType
+        public Microsoft.Data.SqlClient.DataClassification.Label Label
+    }
+}
+```
+
+### UTF-8 Support
+
+UTF-8 support does not require any application code changes. These SqlClient changes simply optimize the communication between the client and server when the server supports UTF-8 and the underlying column collation is UTF-8. See the UTF-8 section under [What's new in SQL Server 2019 preview](https://docs.microsoft.com/en-us/sql/sql-server/what-s-new-in-sql-server-ver15?view=sqlallproducts-allversions#utf-8-support-ctp-23).
```
Comment (peterbae) [inline release-notes/1.0/1.0.19189.1-Preview.md]: I think for the link here and the one below, you want to remove /en-us/ from the URLs.

File: release-notes/1.0/1.0.19189.1-Preview.md [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+* .NET Framework 4.6+
+* .NET Core 2.1+ (Windows x86, Windows x64, Linux, macOS)
+* .NET Standard 2.0+ (Windows x86, Windows x64, Linux, macOS)
+
+### Dependencies
+
+#### .NET Framework
+
+* System.Data.Common 4.3.0
+* Microsoft.Data.SqlClient.SNI
```
Comment (karinazhou) [inline release-notes/1.0/1.0.19189.1-Preview.md]: Do we also need the version for this one?

File: release-notes/1.0/1.0.19189.1-Preview.md (hunk 1 above)
Comment (karinazhou) [reply release-notes/1.0/1.0.19189.1-Preview.md]: I see this in the previous two release notes too. Don't we have solved this by splitting SNI native code from our driver's code?

File: release-notes/1.0/1.0.19189.1-Preview.md (hunk 3 above)
Comment (cheenamalhotra) [reply release-notes/1.0/1.0.19189.1-Preview.md]: This is done intentionally as a sub-list.

File: release-notes/1.0/1.0.19189.1-Preview.md (hunk 2 above)
Comment (cheenamalhotra) [reply release-notes/1.0/1.0.19189.1-Preview.md]: Intentional sub-list

File: release-notes/1.0/1.0.19189.1-Preview.md (hunk 5 above)
Comment (cheenamalhotra) [reply release-notes/1.0/1.0.19189.1-Preview.md]: Yes, its the preview version we released. I can add it.

File: release-notes/1.0/1.0.19128.1-Preview.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+* [Data Classification](#data-classification) - Available in Azure SQL Database and Microsoft SQL Server 2019 since CTP 2.0.
+* [UTF-8 support](#utf-8-support) - Available in Microsoft SQL Server SQL Server 2019 since CTP 2.3.
+* [Always Encrypted with Enclaves](#always-encrypted-with-enclaves) - Always Encrypted is available in Microsoft SQL Server 2016 and higher. Enclave support was introduced in Microsoft Sql Server 2019 CTP 2.0.
+
+### Fixed
+
+.NET Core
+
+* Always Encrypted APIs do not show up in Intellisense
+* Fixed in 1.0.19123.2
```
Comment (karinazhou) [inline release-notes/1.0/1.0.19128.1-Preview.md]: Do we still need to keep fixes from 1.0.19123.2 in this release?

File: release-notes/1.0/1.0.19128.1-Preview.md [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+  * XEvents are arriving as expected, but with 1 XEvent delay [CoreFx PR 36015](https://github.com/dotnet/corefx/pull/36015)
+  * Fix passing null for credentials in SqlConnectionPoolKey [CoreFx PR 34268](https://github.com/dotnet/corefx/pull/34268)
+  * Potential race condition in SqlConnectionString.cs [CoreFx Issue 30659](https://github.com/dotnet/corefx/issues/30650)
+
+### Changes
+
+.NET Core performance improvements
+
+* Changed in 1.0.19123.2
```
Comment (karinazhou) [inline release-notes/1.0/1.0.19128.1-Preview.md]: Do we still need to keep changes from 1.0.19123.2 in this release?

File: release-notes/1.0/1.0.19128.1-Preview.md (hunk 6 above)
Comment (cheenamalhotra) [reply release-notes/1.0/1.0.19128.1-Preview.md]: I guess David wanted to keep them included - useful for GA maybe?

File: release-notes/1.0/1.0.19189.1-Preview.md [hunk 8]
```diff
@@ -0,0 +1,159 @@
...
+* When targeting **.NET Framework** and platform **AnyCPU** or **x86**, you may see a build warning about processor architecture mismatch: *"Warning: There was a mismatch between the processor architecture of the project being built "MSIL" and the processor architecture of the reference "Microsoft.Data.SqlClient...*"
+
+.NET Framework
+
+* Fixed an issue when a pooled connection was switched between databases supporting UTF-8 which resulted in a mismatch between encoding and code page.
+
+.NET Core
+
+* When doing a hard copy of a sql parameter, the MetaType was not copied if the value of the parameter was not defined. This resulted in the wrong SqlType of the parameter.
+* Fixed in 1.0.19123.2-Preview
```
Comment (karinazhou) [inline release-notes/1.0/1.0.19189.1-Preview.md]: Do we still need to keep fixes from 1.0.19123.2 in this release?

File: release-notes/1.0/1.0.19189.1-Preview.md [hunk 9]
```diff
@@ -0,0 +1,159 @@
...
+
+.NET Core changes
+
+* Enable BeginExecuteReader methods on SqlCommand from [CoreFx PR 33660](https://github.com/dotnet/corefx/pull/33660)
+
+.NET Core performance improvements
+
+* Avoid unnecessary Span to ReadOnlySpan casts from [CoreFx PR 34646](https://github.com/dotnet/corefx/pull/34646)
+* SqlClient environment change optimization from [CoreFx PR 34573](https://github.com/dotnet/corefx/pull/34573)
+* Changes in 1.0.19123.2-Preview
```
Comment (karinazhou) [inline release-notes/1.0/1.0.19189.1-Preview.md]: Do we still need to keep changes from 1.0.19123.2 in this release?

File: release-notes/1.0/1.0.19128.1-Preview.md (hunk 6 above)
Comment (karinazhou) [reply release-notes/1.0/1.0.19128.1-Preview.md]: OK let's keep it then.

File: release-notes/1.0/1.0.19128.1-Preview.md (hunk 7 above)
Comment (karinazhou) [reply release-notes/1.0/1.0.19128.1-Preview.md]: OK. Let's keep the old release notes untouched.

**Pull request merged**